packages/berlin/src/pages/Account.tsx

```diff
@@ -202,24 +202,26 @@ function AccountForm({
   });
 
   const onSubmit = (value: typeof initialUser) => {
-    if (user && user.id) {
-      mutateUserData({
-        userId: user.id,
-        username: value.username,
-        email: value.email,
-        groupIds: [value.group],
-        userAttributes: {
-          ...value.userAttributes,
-          credentialsGroup: JSON.stringify(value.userAttributes?.credentialsGroup),
-          publications: JSON.stringify(value.userAttributes?.publications),
-          contributions: JSON.stringify(value.userAttributes?.contributions),
-        },
-      });
+    if (isValid) {
```

Comment:
This might me joined as a single if `if (isValid && user && user.id)`

Reply:
Good point! Fixed.